packages/dashboard/src/dashboard-actions/reverse-actions/reverseMove.ts

```diff
@@ -1,13 +1,14 @@
 import { MoveAction } from '../../dashboard-actions/actions';
 
 export const reverseMove = (moveAction: MoveAction): MoveAction => {
-  if (typeof moveAction.payload.prevPosition != 'undefined') {
+  if (typeof moveAction.payload.prevPosition != 'undefined' && typeof moveAction.payload.widgetIds != 'undefined') {
```

Comment:
why not just say `moveAction.payload.prevPosition != null` instead of `typeof`? more idiomatic way or doing this

Reply:
i tried that and i got an error of sorts, but now it's fixed